app/src/main/java/com/example/returnpals/composetools/PackagesUI.kt

```diff
@@ -55,19 +55,20 @@ import com.example.returnpals.ScheduleReturn
 ////////////////////
 
 @Composable
-fun ScheduleReturn.PackagesUI(
-    packages: List<PackageInfo> = listOf(),
-    navController: NavController? = null,
-    onAddLabel: (PackageInfo) -> Unit = {},
-    onRemoveLabel: (Long) -> Unit = {},
+fun PackagesUI(
+    packages: List<PackageInfo>,
+    onAddLabel: (PackageInfo) -> Unit,
+    onRemoveLabel: (Long) -> Unit,
+    onClickNext: () -> Unit,
```

Comment:
Looks good with the PackagesUI State. Looks like the next and back buttons would go to another State and higher-level navigation can handle that now.